test/views/declaration/AdditionalDeclarationTypeViewSpec.scala

```diff
@@ -50,6 +51,15 @@ class AdditionalDeclarationTypeViewSpec extends UnitViewSpec with CommonMessages
           backButton must haveHref(DeclarationChoiceController.displayPage)
         }
 
+        "hide the expected 'Back' button" in {
```

Comment:
Need to specify that this is testing the situation where the draft dec is an amendment draft